### PR TITLE
Hack-y fix to 404 on gh-pages

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/0qwW70t6/270-indexhtml-404

## Description

- Hack to stop 404. we will probably fix this in a different way next week.
-

@neontribe/sea-map

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ script:
 - parcel build src/index.html --public-url ./
 before_deploy:
 - cp robots-nofollow.txt dist/robots.txt
+- ln -s dist/index.html dist/404.html
```
Comment (Nikomus) [inline .travis.yml:12]: Are the source/target the right way around?

Comment (katjam) [reply .travis.yml:12]: I believe so. the way I remember it is 'real thing first' since 2nd param is optional.

Comment (Nikomus) [reply .travis.yml:12]: OK, just checking!

**Review by Nikomus — APPROVED** [review]
🔗 💥

**Pull request merged**